src/analytics/api/utils/utils.py

```
@@ -0,0 +1,50 @@
+            file_name(str): The name of the schema file to load.
+
+        Returns:
+            Dict: The parsed JSON content of the schema file.
+
+        Raises:
+            FileNotFoundError: If the file is not found in either the "schema" directory or the given path.
+        """
+        current_file = Path(__file__).parent.parent.parent
+        file_name_path = f"schemas/files/{file_name}"
+        try:
+            file_json = open(os.path.join(current_file, file_name_path))
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)